variables.tf

```diff
@@ -438,6 +382,81 @@ variable "kms_rotation_interval_month" {
+  description = "Name of the CBR zone which represents IBM Cloud billing. See [What are CBRs?](https://cloud.ibm.com/docs/account?topic=account-context-restrictions-whatis)"
+  default     = "billing-reports-bucket-writer"
+  validation {
+    condition     = can(regex("^[a-zA-Z0-9 -_]{1,128}$", var.cbr_billing_zone_name))
+    error_message = "Invalid `cbr_billing_zone_name`: value must meet the following regular expression /^[a-zA-Z0-9 -_]+$/ and have length > 1 and < 128"
+  }
+}
+
+variable "cbr_cloudability_zone_name" {
+  type        = string
+  description = "Name of the CBR zone which represents IBM Cloudability. See [What are CBRs?](https://cloud.ibm.com/docs/account?topic=account-context-restrictions-whatis)"
+  default     = "cldy-reports-bucket-reader"
+  validation {
+    condition     = can(regex("^[a-zA-Z0-9 -_]{1,128}$", var.cbr_cloudability_zone_name))
+    error_message = "Invalid `cbr_cloudability_zone_name`: value must meet the following regular expression /^[a-zA-Z0-9 -_]+$/ and have length > 1 and < 128"
+  }
+}
+
+variable "cbr_cos_zone_name" {
+  type        = string
+  description = "Name of the CBR zone which represents Cloud Object Storage service. See [What are CBRs?](https://cloud.ibm.com/docs/account?topic=account-context-restrictions-whatis)"
+  default     = "cldy-reports-object-storage"
+  validation {
+    condition     = can(regex("^[a-zA-Z0-9 -_]{1,128}$", var.cbr_cos_zone_name))
+    error_message = "Invalid `cbr_cloudability_zone_name`: value must meet the following regular expression /^[a-zA-Z0-9 -_]+$/ and have length > 1 and < 128"
+  }
+}
+
+variable "cbr_schematics_zone_name" {
+  type        = string
+  description = "Name of the CBR zone which represents Schematics. The schematics zone allows Projects to access and manage the Object Storage bucket."
+  default     = "schematics-reports-bucket-management"
+  validation {
+    condition     = can(regex("^[a-zA-Z0-9 -_]{1,128}$", var.cbr_schematics_zone_name))
+    error_message = "Invalid `cbr_schematics_zone_name`: value must meet the following regular expression /^[a-zA-Z0-9 -_]+$/ and have length > 1 and < 128"
+  }
+}
+
+variable "cbr_additional_zone_name" {
+  type        = string
+  description = "Name of the CBR zone that corresponds to the ip address range set in `additional_allowed_cbr_bucket_ip_addresses`."
+  default     = "additional-billing-reports-bucket-access"
+  validation {
+    condition     = can(regex("^[a-zA-Z0-9 -_]{1,128}$", var.cbr_additional_zone_name))
+    error_message = "Invalid `cbr_additional_zone_name`: value must meet the following regular expression /^[a-zA-Z0-9 -_]+$/ and have length > 1 and < 128"
+  }
+}
+
+variable "additional_allowed_cbr_bucket_ip_addresses" {
+  type        = list(string)
+  description = "A list of CBR zone IP addresses, which are permitted to access the bucket.  This zone typically represents the IP addresses for your company or workstation to allow access to view the contents of the bucket."
```

Comment:
would be nice to mention in the description that this can be a list of IP's or a range of IP's